app/controllers/quick_reservations_controller.rb

```diff
@@ -133,6 +133,8 @@ def get_startable_reservation
     ongoing = reservations.ongoing.first
 
     @reservation = ongoing || startable
+
+    @reservation&.order_detail&.problem? ? nil : @reservation
```

Comment:
I think we want:
```suggestion
    @reservation = nil if @reservation&.order_detail&.problem?
```
... right?
Can you call this `set_startable_reservation` instead of `get_startable_reservation`?